autoplex/data/rss/utils.py

```diff
@@ -92,15 +115,12 @@ class HookeanRepulsion(FixConstraint):
     that the constraint will be either soft enough (e.g. non-exploding in MD) or
     strong enough (to avoid overlaps) for all spring constants and distances.
 
-
     Adapted from:
```

Comment:
A general comment: We need to think of an alternative way to document this; the current method does not seem to render correctly in API documentation. 

https://autoatml.github.io/autoplex/reference/autoplex.data.rss.utils.HookeanRepulsion.html#autoplex.data.rss.utils.HookeanRepulsion

Reply:
@MorrowChem, do you have any ideas for this?

Reply:
I just saw, `References` can be section header, that could be used for such cases I think.

Maybe have a look at possible header options here and see what you seem is the best fit https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#docstring-sections